### PR TITLE
Fixed formatting, adding new logging sources

### PR DESCRIPTION
**Pull request by soggiest**
-Fixed the formatting around "TD_AGENT_USER" and "TD_AGENT_GROUP" between lines 55 and 65
-Added logging config file for PaceMaker
-Added logging config file for Corosync

### CODE REVIEW

File: playbooks/Installation/logging_with_fluentd.adoc [hunk 1]
```diff
@@ -131,6 +136,61 @@ cat << EOF > /etc/td-agent/config.d/kubernetes.conf
 EOF
 ----
 
+OSE does not have logs of it's own, however if you are using an HA OSE environment there are some services that are useful to provide centralized logging.
```
Comment (etsauer) [inline playbooks/Installation/logging_with_fluentd.adoc]: Can you change this to say something like openshift doesn't log to a unique file, but instead to /var/log/whatever (messages, i think). This makes it sound like openshift components don't log anything.

**Comment by soggiest** [comment]
Sure, I wasn't aware that it did log to /var/log/messages.

**Comment by etsauer** [comment]
@soggiest so OpenShift logs to journald, but I believe that journald is configured out of the box to write to /var/log/messages. I'm not 100% on that. @oybed do you know?

**Comment by oybed** [comment]
@soggiest @etsauer rsyslog is capable of pulling info from journald and log it to its "normal" files. I believe the default configuration is this behavior and hence you see logs in the typical syslog files. 

BTW: there's a change coming around this in newer versions of systemd (starting with 216), but don't believe a typical RHEL 7.1 installation is based on that version of systemd yet (I believe it's 208 or something like that).  Here are some resources with more info:

https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/s1-interaction_of_rsyslog_and_journal.html

http://lists.freedesktop.org/archives/systemd-devel/2014-August/022295.html#journald

**Comment by etsauer** [comment]
@oybed so would you agree with my suggested change to @soggiest's doc, or would you say something differently.

**Comment by soggiest** [comment]
I found evidence of OSE logs in /var/log/messages so I changed the wording of my commit per your request @etsauer

File: playbooks/Installation/logging_with_fluentd.adoc [hunk 2]
```diff
@@ -131,6 +136,63 @@ cat << EOF > /etc/td-agent/config.d/kubernetes.conf
...
+    time_slice_format %Y%m%d
+    time_slice_wait 10m
+    time_format %Y%m%dT%H%M%S%z
+    compress gzip
+    utc
+  </match>
+
+----
+
+TODO: Add EXEC logging and parsing for Openshift logging in /var/log/messages.
```
Comment (etsauer) [inline playbooks/Installation/logging_with_fluentd.adoc]: Instead of the TODO: block, let's use the Feedback Request from the style guide

**Pull request merged**